assets/src/components/v2/cr_departures.tsx

```diff
@@ -1,20 +1,78 @@
 import BaseDepartureTime from "Components/eink/base_departure_time";
 import Arrow from "Components/solari/arrow";
-import React from "react";
+import useInterval from "Hooks/use_interval";
+import React, { useState } from "react";
 import { classWithModifier, imagePath } from "Util/util";
 import CRIcon from "./bundled_svg/cr_icon";
 import Free from "./bundled_svg/free";
 import ClockIcon from "./clock_icon";
 
+interface Departure {
+  arrow: string,
+  headsign: {
+    headsign: string,
+    variation: string
+  },
+  time: any,
+  track_number: number
+}
 interface CRDeparturesProps {
-  departures: any,
+  departures: Departure[],
   destination: string,
   time_to_destination: string,
   show_via_headsigns_message: boolean
 }
 
+const DeparturesTable: React.ComponentType<any> = (props) => {
+  const {departures} = props
+
+  let [headsignPageOne, setHeadsignPageOne] = useState(true);
+
+  useInterval(() => {
+    setHeadsignPageOne(!headsignPageOne);
+  }, 2000);
+
+  return (
+    <table>
+      <tbody>
+        <tr>
+          <td className="wayfinding-arrow-column"></td>
+          <td className="headsign-column">
+            <div className="table-header__english">Upcoming departures</div>
+            <div className="table-header__spanish">Próximas salidas</div>
+          </td>
+          <td className="arrival-column"></td>
+          <td className="track-column">
+            <div className="table-header__english">Track</div>
+            <div className="table-header__spanish">Pista</div>
+          </td>
+        </tr>
+        { departures.slice(0, 3).map((departure, i) => {
+            return (
+              <tr key={i}>
```

Comment:
Does `departure` have an ID field, or any way to identify it uniquely by its values, in a way that we can track it across data refreshes? Please use that as the `key` instead. You may need to add the departure ID to the values sent back from the server by `serialize/1`. Using array indices as `key`s causes issues/incorrect updates when the array contents change.